index.html

```diff
@@ -59,9 +74,19 @@ <h2 class="nomargin">Experience</h2>
             <li>Lesson 1.4: JavaScript Arrays</li>
             <li>Lesson 1.5: JavaScript Objects</li>
         </ul>
+    </div> -->
+
+    <div class="container experience" id="container">
```

Comment:
I think the goal of the assignment was to make this into a grid similar to what is happening here instead of a grid: https://www.theodinproject.com/lessons/foundations-introduction-to-flexbox#lets-flex